src/content/docs/en/guides/fonts.mdx

```diff
@@ -103,30 +103,21 @@ See the [Fontsource guide to preloading fonts](https://fontsource.org/docs/getti
 
 ## Register fonts in Tailwind
 
-If you are using the [Tailwind integration](/en/guides/integrations-guide/tailwind/), you can use either of the previous methods on this page to install your font, with some modification. You can either add an [`@font-face` statement for a local font](#using-a-local-font-file) or use [Fontsource's `import` strategy](#using-fontsource) to install your font.
+If you are using [Tailwind](/en/guides/styling/#tailwind), you can use either of the previous methods on this page to install your font, with some modification. You can either add an [`@font-face` statement for a local font](#using-a-local-font-file) or use [Fontsource's `import` strategy](#using-fontsource) to install your font.
```

Comment:
here it should be modifications (plural)
```suggestion
If you are using [Tailwind](/en/guides/styling/#tailwind), you can use either of the previous methods on this page to install your font, with some modifications. You can either add an [`@font-face` statement for a local font](#using-a-local-font-file) or use [Fontsource's `import` strategy](#using-fontsource) to install your font.
```

Reply:
I don't object, but I *think* either actually works here!